modules/table/src/test/java/org/apache/ignite/internal/table/distributed/replication/ZonePartitionReplicaListenerTest.java

```diff
@@ -1466,6 +1464,11 @@ void writeIntentSwitchForCompactedCatalogTimestampWorks(boolean commit) {
         HybridTimestamp beginTs = beginTimestamp(txId);
         HybridTimestamp commitTs = clock.now();
 
+        // We have to force push clock forward because we will invoke listener directly bypass ReplicaManager or MessageService, so clock
```

Comment:
```suggestion
        // We have to force push clock forward because we will invoke listener directly bypassing ReplicaManager or MessageService, so clock
```

Reply:
Applied.